### DISCUSSION TITLE
Definition of Vertical Velocity in Parcels

### DISCUSSION THREAD

**Discussion by BenOwusu084** [Parcels-code/Parcels#1465] [General]
Hi, Please I would like to know how the vertical velocity is defined in the Ocean Parcels model. For example which direction (upward or downward / + or -) does it mean in the model if I give a negative value (eg. -1e-6 m/s) to the model.

**Comment by erikvansebille** [comment]
Dear @BenOwusu084, the default `AdvectionRK4_3D` kernel solves the equation `dz = w*dt`. 

So the important thing is that the direction of `w` and of `z` are the same. In other words, if `z` increases moving downward, then the advection kernel assumes that `w` is positive _downwards_.

If that is not the case (e.g. `z` is positive downwards but `w` is positive upwards), you can use `fieldset.W.set_scaling_factor(-1.)` to flip the sign of `W`

**Reply by BenOwusu084** [reply]
Thank a lot for the swift reply @erikvansebille . I asked this because I run the model to simulate the trajectories of particles initialized in the North Atlantic subtropical gyre region for a year but I realized the particles still remain at the surface when they converge towards the center of the gyre (which in actual sense they should be sink due to downwelling at the center of the gyre). I will perhaps check the vertical velocity data of the ocean field data.
